packages/docs-ui/README.md

```diff
@@ -11,6 +11,7 @@ npm install @braid-design-system/docs-ui
 
 - [`LinkableHeading`](#linkableheading)
 - [`MenuButton`](#menubutton)
+- [`SideNavigationGroup`](#sidenavigationgroup)
```

Comment:
The readme still references `SideNavigationGroup` instead of `SideNavigationSection`.

Reply:
ahh whoops, nice catch!